e2e/tests/transfer/authz_test.go

```diff
@@ -22,34 +21,31 @@ import (
 	ibcerrors "github.com/cosmos/ibc-go/v8/modules/core/errors"
 )
 
-func TestAuthzTransferTestSuite(t *testing.T) {
-	testifysuite.Run(t, new(AuthzTransferTestSuite))
-}
-
-type AuthzTransferTestSuite struct {
-	testsuite.E2ETestSuite
-}
+func (s *TransferTestSuite) TestAuthz_MsgTransfer_Succeeds() {
```

Comment:
little confused as to the removal of the `TestAuthzTransferTestSuite` fn.

One limitation to the tests that you may not have been aware of is that we need each file to have a single entrypoint (e.g. TestAuthzTransferTestSuite) and the test suite defined in the same file along side it. 

Was this just an optimization to run more transfer tests using the same 2 chains? If this isn't required, we should revert this.

I tried to run `TestAuthz_MsgTransfer_Succeeds` locally and got `testing: warning: no tests to run` 

I believe this is because the test isn't being picked up correctly due to the limitation I mentioned.

Reply:
We can specify entrypoints for them like: 
`make e2e-test test=TestAuthz_MsgTransfer_Succeeds entrypoint=TransferTestSuite `

Reply:
It's true that this is just to optimize the amount of resources generated.

Reply:
this is really limited by the entrypoint recognition functionality. I will revert them!